WordPress/build.gradle

```diff
@@ -137,6 +139,8 @@ android {
         jalapeno { // Pre-Alpha version, used for PR builds, can be installed along release, alpha, beta, dev versions
             applicationIdSuffix ".prealpha"
             dimension "buildType"
+            // Used for development of editor onboarding, may be remove once finished
+            buildConfigField "boolean", "ENABLE_EDITOR_ONBOARDING", "true"
```

Comment:
> Instead of adding a new `buildConfigField`, did you consider instead just checking if it's a debug build (`BuilcConfig.DEBUG` iirc)? Might be simpler. 

I had not considered that, solely out of ignorance. It does seem leveraging `BuildConfig.DEBUG` would achieve what I want (enabling the feature for development purposes) in a simpler manner. Thanks for the idea!

Reply:
And fwiw, verified that [Japapeno flavored builds for the installable builds are also `debug` variants](https://github.com/wordpress-mobile/WordPress-Android/blob/c7349f6734c230311bf0a1983f7b57ee40286b71/.circleci/config.yml#L130) so yeah, just checking for `BuildConfig.DEBUG` should be enough both for dev build as well as CI-built installable builds.